erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -49,16 +49,17 @@ def validate_payment_request_amount(self):
 			get_existing_payment_request_amount(self.reference_doctype, self.reference_name)
 		)
 
-		ref_doc = frappe.get_doc(self.reference_doctype, self.reference_name)
-		if not hasattr(ref_doc, "order_type") or getattr(ref_doc, "order_type") != "Shopping Cart":
-			ref_amount = get_amount(ref_doc, self.payment_account)
+		if existing_payment_request_amount:
```

Comment:
Suspect this is a conflict merge issue because these changes were done a long time ago and the same code has been changed by other people. It's going to be a while until I can look at it again